pandas/core/arrays/masked.py

```diff
@@ -1097,6 +1108,30 @@ def _wrap_reduction_result(self, name: str, result, skipna, **kwargs):
             return self._maybe_mask_result(result, mask)
         return result
 
+    def _wrap_min_count_reduction_result(
+        self, name: str, result, skipna, min_count, **kwargs
+    ):
+        if min_count == 0 and isinstance(result, np.ndarray):
+            return self._maybe_mask_result(result, np.zeros(1, dtype=bool))
+        return self._wrap_reduction_result(name, result, skipna, **kwargs)
+
+    def _wrap_na_result(self, name):
+        mask = np.ones(1, dtype=bool)
+
+        if is_float_dtype(self.dtype):
```

Comment:
any chance of avoiding these?  weve been moving away from them for perf reasons

Reply:
Ok.